menu/translations/lxqt-leave_cs.directory

```diff
@@ -1,2 +1 @@
-Comment[cs]=Opustit sezení
```

Comment:
Why the "Comment" removal?

Reply:
Because it's not shown in the menu.
I think in weblate is needed a screenshot and explaining.